fusor-ember-cli/app/controllers/deployment-new/satellite/configure-environment.js

```diff
@@ -36,15 +36,23 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, NeedsDeploymen
     selectEnvironment(environment) {
       this.set('showAlertMessage', false);
       this.set('selectedEnvironment', environment);
-      return this.get('deploymentNewController.model').set('lifecycle_environment', environment);
+      this.get('deploymentNewController.model').set('lifecycle_environment', environment);
+      this.get('deploymentNewController').set('errorMsg', null);
+      this.set('errorMsg', null);
     },
 
     createEnvironment(fields_env) {
       var self = this;
+      this.set('showAlertMessage', false);
+      this.set('errorMsg', null);
```

Comment:
this line is the actual fix of the BZ to remove the error message. The rest is refactoring and cleanup.